dev/com.ibm.ws.jaxrs.2.1.appSecurity/bnd.bnd

```diff
@@ -0,0 +1,42 @@
+#*******************************************************************************
+# Copyright (c) 2022 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.0
+
+Bundle-SymbolicName: com.ibm.ws.jaxrs.2.1.appSecurity
+
+-dsannotations: \
+  com.ibm.ws.jaxrs21.providers.security.SecurityAnnoProviderRegister
+
+Export-Package: \
+  com.ibm.ws.jaxrs21.security;thread-context=true
+
+Import-Package: \
+  *
```

Comment:
Is there any downside to relying on ```*``` ? In our other bundles we explicitly import stuff (or explicitly prevent imports) and then have ```*`` as a catch all anyway.

Reply:
it depends on if you need to do special rules for the versioning like with version=!

Reply:
special ranges also would be something that would require it.

Reply:
I don't think I do